_test/test_queries.go

```diff
@@ -93,11 +94,23 @@ func testChannel(ctx context.Context, t *testing.T, src, dst *relayer.Chain, cha
 
 	chans, err := src.ChainProvider.QueryChannels(ctx)
 	require.NoError(t, err)
-	require.Equal(t, 1, len(chans))
-	require.Equal(t, chans[0].Ordering.String(), "ORDER_UNORDERED")
-	require.Equal(t, chans[0].State.String(), "STATE_OPEN")
-	require.Equal(t, chans[0].Counterparty.ChannelId, channelID)
-	require.Equal(t, chans[0].Counterparty.GetPortID(), portID)
+	require.GreaterOrEqual(t, len(chans), 1)
+
+	var channel *chantypes.IdentifiedChannel
+	for _, ch := range chans {
+		if ch.Counterparty.ChannelId == channelID && ch.Counterparty.PortId == portID {
+			channel = ch
+			break
+		}
+	}
+
+	require.NotNil(t, channel)
+
+	require.Equal(t, channel.Ordering.String(), "ORDER_UNORDERED")
+	require.True(t,
```

Comment:
I think this needs to be `require.Truef` since the message is a formatted string.